client/src/components/Homepage/index.jsx

```diff
@@ -0,0 +1,32 @@
+import React, { Component, Fragment } from 'react';
+import Navbar from '../navbar/Navbar';
+import './style.css';
+import MainSidebar from '../MainSidebar';
+
+class Home extends Component {
+  render() {
+    return (
+      <Fragment>
+        <MainSidebar selected="tickets" />
+        <Navbar
+          className="homeNav"
+          children={[
+            <div className="statusdiv">
+              <div className={`pendingDiv ${this.props.pendingClass}`}>
+                <h4 className="status">Pending</h4>
+                <div id="circle">10</div>
```

Comment:
@Anies12 I think you should switch from 'id' to 'className', since there is only supposed to be one element with a given 'id' per page (and there are two in this component).

For this reason, I rarely use ids, myself.